doc/api-docs/conf.py

```diff
@@ -12,33 +12,26 @@
 # All configuration values have a default; values that are commented out
 # serve to show the default.
 
+from datetime import datetime
 import importlib
 import itertools
 import os
 import pkgutil
 import subprocess
 import sys
-
-from datetime import datetime
-from sphinx.ext import apidoc
 from unittest.mock import MagicMock
 
+import natcap.invest
+from sphinx.ext import apidoc
```

Comment:
Could you just check these imports and make sure that we aren't importing anything we no longer need?

Reply:
You're right, a couple of them aren't needed any more